code/drasil-theory/Theory/Drasil/MultiDefn.hs

```diff
@@ -50,6 +50,10 @@ instance Definition    MultiDefn where defn     = rDesc
 -- | The complete Relation of a MultiDefn is defined as the quantity and the related expressions being equal
 --   e.g., `q $= a $= b $= ... $= z`
 instance ExprRelat     MultiDefn where relat q  = sy q $= foldr1 ($=) (NE.map (^. expr) (q ^. rvs))
+-- TODO: How do we want to display MultiDefns?
+instance Display       MultiDefn where
+  -- toDispExpr md = multiExpr $ map (^. expr) $ NE.toList $ md ^. rvs
+  toDispExpr = toDispExpr . relat  -- this one at least doesn't break stable (yet?)
```

Comment:
I think MultiDefns are the hackiest thing I left in before PR.

I'm not quite sure how we'd want to display, so I leave it as it is.

For example, if we had `X` which can be defined by `Y` or `Z`.

Do we want to display it as `X = Y = Z` or as:

```
(i) X = Y
(ii) X = Z
```

or anything else similar?

Reply:
Can we in fact leave it as using 'and' in this PR?

Reply:
Ah, yes, this one currently displays in the `a $= b $= ... $= z` format to be clear. But, of course, we can leave it as it is.